app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -273,9 +280,16 @@ public void onConfigurationChanged(@NonNull Configuration newConfig) {
 
     @Override
     public boolean onMenuItemClick(MenuItem item) {
-        if (item.getItemId() == R.id.action_search) {
-            item.getActionView().post(() -> iconTintManager.updateTint());
+        switch (item.getItemId()) {
```

Comment:
Please use `if` statements. IDs are no longer guaranteed to be constant.

Reply:
I don't think I changed something here (except adding the toolbar DialogSetting).
There are numerous occasions where ids are used with switch, shouldn't that be addressed in it's own PR?

Reply:
It was an `if` (that only checks search) before and is now a `switch` that checks both search and dialog.

> There are numerous occasions where ids are used with switch, shouldn't that be addressed in it's own separate PR?

I don't really like PRs that touch dozens of files at once because it makes it harder to use things like `git bisect`. New code (and changes that touch the lines anyway) should use `if` and we will deal with the remaining `switch` clauses if it really breaks in the future :)